openforcefield/topology/molecule.py

```diff
@@ -1826,14 +1826,21 @@ def __eq__(self, other):
         # TODO the doc string did not match the previous function what matching should this method do?
         return Molecule.are_isomorphic(self, other, return_atom_map=False)[0]
 
-    def to_smiles(self, toolkit_registry=GLOBAL_TOOLKIT_REGISTRY):
+    def to_smiles(self, isomeric=True, explicit_hydrogens=True, mapped=False, toolkit_registry=GLOBAL_TOOLKIT_REGISTRY):
         """
         Return a canonical isomeric SMILES representation of the current molecule
 
         .. note :: RDKit and OpenEye versions will not necessarily return the same representation.
 
         Parameters
         ----------
+        isomeric: bool optional, default= True
+            return an isometric smiles, requires all stereochemistry to be defined
+        explicit_hydrogens: bool optional, default=True
+            return a smiles string containing all hydrogens explicitly
+        mapped: bool optional, default=False
+            return a explicit hydrogen mapped smiles, the atoms to be mapped can be controlled by supplying an
+            atom map into the properties dictionary
```

Comment:
Same comment as above about providing more info about how `offmol.properties['atom_map']` is handled -- The same docstring can just be copied into here.